liblepton/src/geda_complex_object.c

```diff
@@ -911,13 +911,17 @@ o_complex_check_symversion(TOPLEVEL* toplevel, OBJECT* object)
   g_return_if_fail (object->complex != NULL);
 
   /* first look on the inside for the symversion= attribute */
-  inside = o_attrib_search_inherited_attribs_by_name (object, "symversion", 0);
+  inside = o_attrib_search_inherited_attribs_by_name (object,
```

Comment:
Same as [this comment](https://github.com/lepton-eda/lepton-eda/pull/95#discussion_r105783257) on #95; I think it would be better to change the prototype for `o_attrib_search_inherited_attribs_by_name()`.